exec/java-exec/src/test/java/org/apache/drill/test/QueryRowSetIterator.java

```diff
@@ -101,7 +103,7 @@ public DirectRowSet next() {
 
   public void printAll() {
     for (DirectRowSet rowSet : this) {
-      rowSet.print();
+      new RowSetStringBuilder(rowSet).writeTo(new OutputStreamWriter(System.out));
```

Comment:
This is *not* a simplification; it exposes too much detail. Better:

```
RowSetFormatter.print(rowSet);
```

Of course, if we go this route, we might as well leave the `print()` method in `RowSet` and have it call the above code.

Just to be clear, the purpose of `print()` is to help us developers visualize what's going on in Drill; it is not for production use. This is important because, if you can't visualize a batch, you'll make many mistakes because you can't tell what's what.